src/custom/hooks/useGetGpApiStatus.ts

```diff
@@ -0,0 +1,39 @@
+import ms from 'ms.macro'
+import { useState, useEffect } from 'react'
+
+type GpQuoteStatus = 'COWSWAP' | 'LEGACY'
+// TODO: use actual API call
+export async function checkGpQuoteApiStatus(): Promise<GpQuoteStatus> {
+  return new Promise((accept) => setTimeout(() => accept('COWSWAP'), 500))
+}
+const GP_QUOTE_STATUS_INTERVAL_TIME = ms`2 hours`
```

Comment:
Why 2 hours?
Nothing wrong with the frequency, just interested in the reasoning

Reply:
was just a time i mentioned, we can deffo agree on sth else